src/main/java/org/mifos/connector/ams/paygops/camel/route/PaygopsRouteBuilder.java

```diff
@@ -92,6 +92,10 @@ public void configure() {
                             exchange.setProperty("accountStatus",accountStatus.ACTIVE.toString());
                             exchange.setProperty("subStatus", "");
                             exchange.setProperty("dfspId", exchange.getProperty("dfspId"));
+                            exchange.setProperty(TRANSACTION_ID, exchange.getProperty(TRANSACTION_ID));
+                            exchange.setProperty("amount", result.getAmount());
+                            exchange.setProperty("currency", result.getCurrency());
+                            exchange.setProperty("msisdn", result.getWallet_msisdn().substring(1));
```

Comment:
Why are we doing this substring without any condition check. look for the character you are trying to remove.